setup.py

```diff
@@ -1,6 +1,29 @@
-from distutils.core import setup
+from setuptools import setup
 
-setup(name='pysize',
-      version='1.0',
-      py_modules=['pysize'],
-      )
\ No newline at end of file
+with open("README.md", "r") as fh:

```

Comment:
For Python 2 compat use `io.open()` with UTF-8 encoding. Also, newer setuptools allow using declarative `setup.cfg`, but that'd require upgrading setuptools in CI.